doc/guides/build/index.rst

```diff
@@ -0,0 +1,132 @@
+.. _build_overview:
+
+Build Overview
+##############
+
+The Zephyr build process can be divided into two main phases: a
+configuration phase (driven by *CMake*) and a build phase (driven by
+*Make* or *Ninja*). This document is describing the build using *Make*
+as example.
+
+Configuration Phase
+*******************
+
+The configuration phase begins when the user invokes *CMake*,
+specifying a source application directory and a board target.
+
+.. figure:: build-config-phase.png
+    :align: center
+    :alt: Zephyr's build configuration phase
+    :figclass: align-center
+    :width: 80%
+
+*CMake* begins by processing the *CMakeLists.txt* file in the application
+directory, which will refer to the *CMakeLists.txt* file in the Zephyr
```

Comment:
since you started in the present tense ( *Cmake* begins ), you should stay in the present tense: 
```
*CMake* begins by processing the *CMakeLists.txt* file in the application
directory, which refers to the *CMakeLists.txt* file in the Zephyr
top-level directory, which in turn refers to *CMakeLists.txt* files
throughout the build tree (directly and indirectly).
```